src/chainer.js

```diff
@@ -8,11 +10,13 @@ function chainer(functions) {
     let result = x;
 
     for (const f of functions) {
-      result = f(result);
+      if (typeof f === 'function') {
+        result = f(result);
+      }
     }
```

Comment:
The chainer function should return another function that takes the input to be transformed by the chain of functions. Currently, it is trying to return a raw value.